.readthedocs.yml

```diff
@@ -10,6 +10,6 @@ formats: all
 
 # Optionally set the version of Python and requirements required to build your docs
 python:
-  version: 3.6
+  version: 3.8
```

Comment:
The [build.image](https://docs.readthedocs.io/en/stable/config-file/v2.html#build-image) docs say that the `latest` builder (the default) has 3.8, so this syncs with the rest of the project.